src/test/java/science/icebreaker/device_availability/DeviceAvailabilityServiceTest.java

```diff
@@ -181,5 +188,42 @@ public void getDeviceAvailabilityOfUser_exists_success() throws AccountCreationE
             this.deviceAvailabilityService.getDeviceAvailability(null, this.account.getId());
 
         assertThat(deviceAvailabilityList.size()).isEqualTo(2);
-    } 
+    }
+
+    @Test
+    public void sendContactRequest_success() {
```

Comment:
The name of this test is a little misleading, isn't it? since it does not really test if an email is sent (which is probably too difficult for us right now) but the case of a missing/wrong configuration.